code/index.html

```diff
@@ -3,28 +3,210 @@
+            <label for="passWord">Choose your password:</label>
+            <input 
+              id="passWord" 
+              type="password"
+              name="passWord"
+              minlength="6"
+              required
+            />
+          </div>
+          <!-- 1 x Set of radio buttons -->
+          <div class="form-div">
+            <p>Do you have a website today?</p>
+            <input 
+              id="webSiteYes" 
+              type="radio"
+              name="Website"
+              value="yes-website"
+            />
+            <label for="webSiteYes">Yes</label>
+            <input 
+              id="webSiteNo" 
+              type="radio"
+              name="Website"
+              value="no-website"
+            />
+            <label for="webSiteNo">No</label>
+          </div>
+          <!-- 1 x Set of checkboxes -->
+          <div class="form-div checkbox">
+              <div class="checkbox-child">
+              <p>Which of the features below would you want to have on your new website?</p>
+                <div>
+              <div class="checkbox-child">
+              <input 
+                id="featuresChoice" 
+                type="checkbox"
+                name="navigationBar"
+              />
+              <label for="featuresChoice">Navigation bar</label>
+              </div>
+              <div class="checkbox-child">
+              <input 
+                id="featuresChoice" 
+                type="checkbox"
+                name="form"
+              />
+            <label for="featuresChoice">A form</label>
+            </div>
+            <div class="checkbox-child">
+            <input 
+              id="featuresChoice" 
+              type="checkbox"
+              name="articles"
+            />
+            <label for="featuresChoice">Articles</label>
+            </div>
+            <div class="checkbox-child">
+            <input 
+              id="featuresChoice" 
+              type="checkbox"
+              name="links"
+            />
+            <label for="featuresChoice">Links</label>
+            </div>
+            <div class="checkbox-child">
+            <input 
+              id="featuresChoice" 
+              type="checkbox"
+              name="video"
+            />
+            <label for="featuresChoice">Video</label>
+            </div>
+            <div class="checkbox-child">
+            <input 
+              id="featuresChoice" 
+              type="checkbox"
+              name="accordion"
+            />
+            <label for="featuresChoice">Accordion</label>
+            </div>
+          </div>
+          <!--1 extra checkbox-->
+          <div class="form-div checkbox">
+            <div class="checkbox-child">
+            <p>I do care about your security!</p>
+            <input 
+              id="termsconditions" 
+              type="checkbox"
+              name="agree"
+              required
+            />
```

Comment:
Well done, this is a feature we see on almost every site where there's a signup form 🙌